ACLWeb.js

```diff
@@ -210,7 +210,7 @@ function getSearchResults(doc, url) {
 		}
 	}
 	else {
-		let papers = ZU.xpath(doc, '//strong/a[contains(@href, "/anthology/")]');
+		let papers = ZU.xpath(doc, '//section[@id="main"]//strong/a[@class="align-middle"]');
```

Comment:
Is there a better selector than that `align-middle` class? If not, that's fine. It just feels a little fragile.

Reply:
Just `a` will work - thanks!